### DISCUSSION TITLE
Can I change the commands used for /town, /nation/, /res?

### DISCUSSION THREAD

**Discussion by Matuchacz** [TownyAdvanced/Towny#7310] [Q&A] [answered]
Can i change prefixes of commands? For example /town create (name) into /familia crate (name)? All functions would stay the same but the command itself would be different.

**Comment by LlmDl** [comment, marked as answer]
This is done via Aliasing using your server's Commands.yml file 
https://bukkit.fandom.com/wiki/Commands.yml
